### PR TITLE
simplify interview question

### PR DESCRIPTION
**Pull request by kyletl**
fyi @michellejzh @johnnosek4 @nackjicholson

### CODE REVIEW

File: energy_analyzer/models.py [hunk 1]
```diff
@@ -37,6 +37,19 @@ class Measure(BaseModel):
...
+    # Implement this function for the at home challenge
+    def get_savings_for_date_range(self, start: datetime, end: datetime) -> Timeseries:
+        """
+        Takes in a start and end date and returns the expected measure savings. For example,
+        when this function is called from `get_past_and_future_year_of_monthly_energy_usage`,
+        it should return timeseries data that matches the shape of the building energy usage
+        data.
+
+        A correct solution will account for whether the measure is active or not during the
+        given time range.
```
Comment (michellejzh) [inline energy_analyzer/models.py:49]: Ah yeah, putting this info here definitely seems like a good move.

File: energy_analyzer/test_biz_logic.py [hunk 2]
```diff
@@ -3,123 +3,118 @@
...
-BUILDINGS = [
-    Building(
-        name="Building 1",
-        measures=[
-            Measure(
-                name="Building 1 - Measure 1",
-                measure_type=MeasureType.SCHEDULING,
-                start=datetime(year=2020, month=6, day=1),
-                end=datetime(year=2021, month=1, day=1),
+building_1 = Building(
```
Comment (michellejzh) [inline energy_analyzer/test_biz_logic.py:12]: If you wanted to be extra nice you could call these `single_measure`, `multiple_measures` or something.

File: energy_analyzer/test_biz_logic.py [hunk 3]
```diff
@@ -3,123 +3,118 @@
...
             result_partial_coverage, result_full_coverage
         ):
-            ts = partial_coverage["timestamp"]
+            ts = partial_coverage.timestamp
             if ts.year == start_of_month.year and ts.month == start_of_month.month:
-                self.assertLess(full_coverage["value"], partial_coverage["value"])
+                self.assertLess(full_coverage.value, partial_coverage.value)
+
+    def test_get_measure_savings_for_date_range(self):
+        # write this test for the at-home challenge
```
Comment (michellejzh) [inline energy_analyzer/test_biz_logic.py:116]: You should make it clear whether this needs to be filled in as part of the take home or not.

**Review by michellejzh — APPROVED** [review]
Seems like an improvement to me! One main comment on the test case but otherwise looks good.

**Review by kibra — APPROVED** [review]

File: energy_analyzer/test_biz_logic.py (hunk 2 above)
Comment (nackjicholson) [reply energy_analyzer/test_biz_logic.py:12]: shhhhhhh, this is a public repo.

**Comment by nackjicholson** [comment]
@kyletl Since you're out tomorrow and I need to be sending this out to folks, I'm merging. Come back and make any additional changes.

**Pull request merged**